Makefile

```diff
@@ -40,6 +40,22 @@ clean: ALWAYS_RUN
 # --- Tests  ------------------------------------
 # -----------------------------------------------
 
+run-local-test: ALWAYS_RUN
+	docker run --rm -it \
+		--env OVERRIDE_HOSTNAME=mail.example.test \
+		--env LOG_LEVEL=trace \
+		--env POSTFIX_INET_PROTOCOLS=ipv4 \
+		--env DOVECOT_INET_PROTOCOLS=ipv4 \
+		--env ENABLE_RSPAMD=1 \
+		--env ENABLE_AMAVIS=0 \
+		--env ENABLE_CLAMAV=0 \
+		--env ENABLE_FAIL2BAN=0 \
+		--env ENABLE_OPENDMARC=0 \
+		--env ENABLE_OPENDKIM=0 \
+		--env ENABLE_POLICYD_SPF=0 \
+		--env TZ=Europe/Berlin \
+		mailserver-testing:ci
+
```

Comment:
Not sure if we want to leave this in this PR, but I frequently used this command during testing..

Reply:
The `TZ` is rather specific, as is the choice to use rspamd (_which eventually becomes redundant once it's default_).

Other than that, since it's not used for running actual test suite it's more of a local dev instance and doesn't really belong in the tests section here?

Most of the env you could provide yourself via `--env-file` arg? Or since this is tailored for you, perhaps an alias on your end works well too?

---

It's probably best to only add this if it meets the needs of others as well, unless `--env-file` and/or shell alias doesn't work well for your needs?